frontend/services/entities/hosts.ts

```diff
@@ -279,19 +259,24 @@ export default {
         osSettings,
       }),
       ...reconcileMutuallyExclusiveHostParams({
```

Comment:
This is the fix as well: `osName`, `osVersionId`, `osVersion`, `bootstrapPackageStatus` did not exist (released bugs)

Reply:
this fix is already in another pr